pages/public_cloud/compute/securing_and_structuring_projects/guide.en-gb.md

```diff
@@ -0,0 +1,139 @@
+* cloud_project_marketing_analytics_dev
+* cloud_project_marketing_automation_qa
+* cloud_project_sales_cpq_staging
+* cloud_project_sales_ecommerce_dev
+
+
+### Step 4: Providing Role-Based Access Control (RBAC) to Public Cloud projects through IAM Policies 
+
+Now that you have identities and projects, you can define the access right for each projects. We recommend to use groups and not individual identities when defining policy since this makes the policy lifecycle management easier. 
+
+In the following example we will use two groups called `finance_developer_group` and `finance_SRE_group` that contains the identities of the users that are developing & the one that are making sure that the finance application in in production in a healthy state.
+
+For this example, we will take a tool used by finance team to make their financial planning and analysis (FPA). In this fictive case, we will have 3 public cloud projects managing the resources of three environments (development, staging and production). The projects are named:
+* `cloud_project_finance_fpa_dev` 
+* `cloud_project_finance_fpa_staging`
+* `cloud_project_finance_fpa_prod`
+
+The following table sums up the access rights we will provide to each groups in the example. The access policies are provided for educational purpose and shall be adapted to your context.
+
+
+| Public Cloud project name | `finance_developer_group` | `finance_SRE_group`
+|------|------|------|
+|`cloud_project_finance_fpa_dev`|read write | no access |
+|`cloud_project_finance_fpa_staging`| read | read write|
+|`cloud_project_finance_fpa_prod`| no access | read write |
+
+
+
+#### Provide Control Panel access
+In order to provide access to OVHCloud control Panel, you need to create a specific policy as described in that [page](/pages/account_and_service_management/account_information/iam-control-panel-access/) and in the `Add user groups`, choose the groups `finance_developer_group` and `finance_SRE_group`.
+![Add user groups](img/Add_user_groups.png){.thumbnail}
+
+Thanks to that policy, the user belonging to those 2 groups will have access to the Control Panel but only with that policy they do not have the right to see / manage any resources. The following chapter adds that. 
+
+#### Provide specific rights to Public Cloud projects
+Let's continue by creating policies in order to enable the access rights tables described above.For that we will create 4 policies summarized below.
+
+
+| Policy name |Identities | Actions |  Resources | 
+|------|------|------|-----|
+|cloud_project_finance_fpa_dev-RW|finance_developer_group|globalWriteAccess|cloud_project_finance_fpa_dev|
+|cloud_project_finance_fpa_staging-RO|finance_developer_group|globalReadAccess|cloud_project_finance_fpa_staging|
+|cloud_project_finance_fpa_staging-RW|finance_SRE_group|globalWriteAccess|cloud_project_finance_fpa_staging|
+|cloud_project_finance_fpa_prod-RW|finance_SRE_group|globalWriteAccess|cloud_project_finance_fpa_prod|
```

Comment:
There seems to be a formatting issue with this table:
https://github.com/ovh/docs/blob/492de153204763868d6bd3de29f85e140417f0f3/pages/public_cloud/compute/securing_and_structuring_projects/guide.en-gb.md#provide-specific-rights-to-public-cloud-projects